Core.cpp

```diff
@@ -45,10 +48,18 @@ int main(int, char**)
 
 		if (bboxes.size() && !flag) {
 			//High Beam detected
+
+			detected_image = frame; //save detected frame;
+
+			sprintf(file,"Image_%d.jpg",count);  //save files with different names using 'count'     
```

Comment:
Went through a few links on stack overflow, and find that sprintf is the standard that everyone follows.. Can experiment with '+' as well.